cwm_worker_operator/deployer.py

```diff
@@ -89,6 +89,8 @@ def deploy_worker(domains_config, deployer_metrics, deployments_manager, worker_
             'CDN_CACHE_NOCACHE_REGEX': '\\.({})$'.format('|'.join(volume_config.cache_exclude_extensions)) if len(volume_config.cache_exclude_extensions) > 0 else '',
             'CDN_CACHE_PROXY_CACHE_VALID_200': '{}m'.format(volume_config.cache_expiry_minutes),
             'CDN_CACHE_PROXY_INACTIVE': '{}m'.format(volume_config.cache_expiry_minutes + 1),
+            'DISABLE_HTTP': 'true' if 'disable_http' in volume_config.protocols_enabled else 'false',
+            'DISABLE_HTTPS': 'true' if 'disable_https' in volume_config.protocols_enabled else 'false',
```

Comment:
these fields should be populated like this:
DISABLE_HTTP: 'true' if 'http' not in protocols_enabled
DISABLE_HTTPS: 'true' if 'https' not in protocols_enabled

Reply:
This won't work. Please see the explanation here: https://stackoverflow.com/a/12199811/7670262.

```python
            'DISABLE_HTTP': 'true' if 'http' not in volume_config.protocols_enabled else 'false',
            'DISABLE_HTTPS': 'true' if 'https' not in volume_config.protocols_enabled else 'false',
```

or,

```
            '{}'.format('http' not in volume_config.protocols_enabled).lower()
            '{}'.format('https' not in volume_config.protocols_enabled).lower()
```

The first one looks more readable.

Reply:
it was just pseudo-code, the important thing is that protocols_enabled should remain as it was